src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,43 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Opening Records in Modal Windows
+
+Record lists have an option to open records in a modal on record click.
+By default, records are opened in a new page.
+When set, this option allows users to inspect specific records, while staying on the original page.
+
+=== Configuring the Option
+
+To enable the option of opening records in a modal, navigate to the page builder, choose the record list block and go to its record list tab. 
+Scroll down to the Records section and you will find the option in the "On record click" select dropdown.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/enabling-record-in-modal.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 0,
+    "y": 0,
+    "w": 1920,
+    "h": 1080
+  },
+  "annotations": [{
```

Comment:
Incorrect -- outline the entire input.

![image](https://user-images.githubusercontent.com/12545711/233564293-7e90493a-d256-4dbc-9444-e843d838016f.png)


correct -- entire input outlined.

![image](https://user-images.githubusercontent.com/12545711/233564488-54c448e7-44fb-4585-b688-b23ab20c4881.png)

**But**, generally, I prefer to have an entire section outlined so let's have it like this please:

![image](https://user-images.githubusercontent.com/12545711/233564722-0bf33b40-7a47-4c42-88b8-80e30077f1e0.png)